src/garage/tf/misc/tensor_utils.py

```diff
@@ -272,12 +278,132 @@ def positive_advs(advs, eps, name=None):
 def discounted_returns(discount, max_len, rewards, name=None):
     with tf.name_scope(name, 'discounted_returns',
                        [discount, max_len, rewards]):
-        gamma = tf.constant(
-            float(discount), dtype=tf.float32, shape=[max_len, 1, 1])
+        gamma = tf.constant(float(discount),
+                            dtype=tf.float32,
+                            shape=[max_len, 1, 1])
         return_filter = tf.math.cumprod(gamma, exclusive=True)
-        rewards_pad = tf.expand_dims(
-            tf.concat([rewards, tf.zeros_like(rewards[:, :-1])], axis=1),
-            axis=2)
-        returns = tf.nn.conv1d(
-            rewards_pad, return_filter, stride=1, padding='VALID')
+        rewards_pad = tf.expand_dims(tf.concat(
+            [rewards, tf.zeros_like(rewards[:, :-1])], axis=1),
+                                     axis=2)
+        returns = tf.nn.conv1d(rewards_pad,
+                               return_filter,
+                               stride=1,
+                               padding='VALID')
+    return returns
+
+
+def discounted_returns_meta_learn(discount, max_len, episode_per_task,
+                                  rewards):
```

Comment:
I really think it shouldn't be necessary to implement this function. Isn't it basically the same as the function above?